src/components/molecules/DatePickerInput/DatePickerInput.tsx

```diff
@@ -24,13 +25,76 @@ export interface DatePickerInputProps {
   placeholder?: string
   disabled?: boolean
   value?: string
+  timePicker?: boolean
```

Comment:
**[blocker]** a lot of duplicated types here. Everything that is defined under DatePickerComponentProps is also defined under DatePickerInputProps, so DatePickerInputProps could just extend DatePickerComponentProps.

Only exception is ariaLabelToUse, but this is not exactly necessary and is also not correctly defined. It is not required by DatePickerComponent, however it is always passed, since ariaLabel is required by DatePickerInput. There is also no need to pass both ariaLabel and ariaLabelToUse here, you will not need both, so just ariaLabel will be enough.